netwire-input.cabal

```diff
@@ -30,7 +30,8 @@ library
   default-extensions:  FunctionalDependencies
                        MultiParamTypeClasses
   exposed-modules:     FRP.Netwire.Input
-  build-depends:       base >= 4.6 && < 4.10,
+  other-modules:       FRP.Netwire.Input.Util
+  build-depends:       base >= 4.6 && < 4.9,
```

Comment:
I thought it was a mistake that you specify <4.10, unless there's something I'm aware of and 4.9 has been fully specified already?